definitions/npm/lodash_v4.x.x/flow_v0.104.x-/lodash_v4.x.x.js

```diff
@@ -200,10 +209,6 @@ declare module "lodash" {
     array: ?Array<T>
   ) => mixed;
   declare type Iteratee<T> = _Iteratee<T> | Object | string;
-  declare type FlatMapIteratee<T, U> =
```

Comment:
this was used for iteratees in functions which accept arrays, so it was renamed to `AFlatMapIteratee`, to make it more consistent with `OFlatMapIteratee` naming style

`| Object` was removed because
1. `Object` is `any`
1. according to docs, iteratee in all flatMap functions should be function, object is not mentioned, and also does not do anything useful if passed